src/Scripts/Game/Features/ID/Licence.c

```diff
@@ -0,0 +1,15 @@
+class Licence
+{
+	protected RTTextureWidget m_wRenderTargetTextureWidget;
+	TextWidget licenceTextWidget;
```

Comment:
```suggestion
	TextWidget licence_TextWidget;
```

According to your previous code i see this format in [EL_IDCardEntity.c](https://github.com/EveronLife/EveronLife/pull/120/files#diff-21310272323b85e30fc25da863e07b70aa6b662936b6f12c67ba5a985d7fa5ba)

Reply:
No, should be as follows.

m_LicenceTextWidget

Reply:
Oh Okay :) Thanks for you reply :D